client/src/components/Admin/Products/DisplayProducts.js

```diff
@@ -55,6 +56,12 @@ import "../css/pagination.css"
   return (
     <div className="main-panel">
    <div className="content-wrapper">
+   <link
```

Comment:
import css, don't add it as links here

Reply:
This is for the fontawesome to get working it wasnt working earlier so tried this

Reply:
did you check Mani's code to see how he added fontawesome?